File: py/1912.py
a = int(input())
maxsum=-1001
sum =0
arr = list(map(int, input().split()))
for i in range(a):
    b =  arr[i]
    sum+=b
    if sum>maxsum:
        maxsum = sum
        pass
    if sum<0:
        sum=0
        pass
    pass

print(maxsum)

File: cpp/1912.cpp
#include <iostream>
using namespace std;
int main()
{
	int c, num;
	int sum = 0, maxsum = -1001;
	cin >> c;
	while (c--)
	{
		cin >> num;
		sum += num;
		if (sum > maxsum)
			maxsum = sum;
		if (sum < 0)
		{
			sum = 0;
		}
	}
	cout << maxsum;
}